pkgs/development/python-modules/authres/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, buildPythonPackage, python }:
+
+let
+  majorversion = "1.1";
+  minorversion = "0";
+in buildPythonPackage rec {
+  pname = "authres";
+  version = "${majorversion}.${minorversion}";
+
+  src = fetchurl {
```

Comment:
Any reason not to use `fetchPypi`?

Reply:
No particular reason. This is the upstream/source. Of course pypi is mirrored, so if you want I can change.

Reply:
Yes, please do. This will facilitate (semi-)automatic upgrades.

Reply:
And it will allow us to get rid of `majorversion` and `minorversion`.